<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/icons/rom_icons.qrc">
    <normaloff>:/new/icons/engineer_mode_128.png</normaloff>:/new/icons/engineer_mode_128.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #EBF0F5;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget {
   background-color: #EBF0F5;
	color: #1E1E1E;
   font-family: 'SF Pro Text';
}

QWidget#centralwidget QPushButton {
	color: #1E1E1E;
	background-color:#99C1F1;
	border-radius: 6px;
	font-size: 12px;
	border: 1px solid #C0C0C0;
	height: 40px;
	min-width: 180px;
	
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignmentFlag::AlignVCenter">
     <widget class="QWidget" name="appBarHorizontalLayout" native="true">
      <property name="styleSheet">
       <string notr="true">border-top-left-radius: 8px; 
border-top-right-radius: 8px;
border-bottom-left-radius: 0px; 
border-bottom-right-radius: 0px;
border: 1px solid #3a474f; 
background-color: #3a474f;
</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;
border: none;
background: none;
text-align: center;
</string>
         </property>
         <property name="text">
          <string>System Settings</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="mainAppHorizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="menuButtonVerticalLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QPushButton" name="basicSettingBtn">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Basic</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="languageSettingBtn">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Language</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="networkSettingBtn">
          <property name="text">
           <string>Network</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="distributionModeBtn">
          <property name="text">
           <string>Distribution Mode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="advertisingSettingBtn">
          <property name="text">
           <string>Advertising</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="versionSettingBtn">
          <property name="text">
           <string>Version</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::NoFrame</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <widget class="QWidget" name="basicPageWidget">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_3">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>576</width>
               <height>519</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="styleSheet">
                 <string notr="true">color: black;</string>
                </property>
                <property name="text">
                 <string>Media Volume</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QPushButton" name="pushButton_7">
                   <property name="styleSheet">
                    <string notr="true">min-width: 50;</string>
                   </property>
                   <property name="text">
                    <string>-</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSlider" name="horizontalSlider">
                   <property name="orientation">
                    <enum>Qt::Orientation::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_8">
                   <property name="styleSheet">
                    <string notr="true">min-width: 50;</string>
                   </property>
                   <property name="text">
                    <string>+</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="styleSheet">
                 <string notr="true">color: black;</string>
                </property>
                <property name="text">
                 <string>Screen Brightness</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_2" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QPushButton" name="pushButton_10">
                   <property name="styleSheet">
                    <string notr="true">min-width: 50;</string>
                   </property>
                   <property name="text">
                    <string>-</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSlider" name="horizontalSlider_2">
                   <property name="orientation">
                    <enum>Qt::Orientation::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_9">
                   <property name="styleSheet">
                    <string notr="true">min-width: 50;</string>
                   </property>
                   <property name="text">
                    <string>+</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="styleSheet">
                 <string notr="true">color: black;</string>
                </property>
                <property name="text">
                 <string>Set Battery Percentage To Charge</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_3" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QPushButton" name="pushButton">
                   <property name="styleSheet">
                    <string notr="true">min-width: 50;
</string>
                   </property>
                   <property name="text">
                    <string>-</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSlider" name="horizontalSlider_3">
                   <property name="orientation">
                    <enum>Qt::Orientation::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_2">
                   <property name="styleSheet">
                    <string notr="true">min-width: 50;</string>
                   </property>
                   <property name="text">
                    <string>+</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Orientation::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="languagePageWidget">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea_2">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_4">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>200</width>
               <height>60</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QPushButton" name="pushButton_11">
                <property name="text">
                 <string>change language</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="networkPageWidget">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea_3">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_5">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>66</width>
               <height>16</height>
              </rect>
             </property>
             <widget class="QPushButton" name="pushButton_13">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>30</y>
                <width>182</width>
                <height>25</height>
               </rect>
              </property>
              <property name="text">
               <string>connect wifi</string>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="distributionModePageWidget">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea_4">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_6">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>66</width>
               <height>16</height>
              </rect>
             </property>
             <widget class="QPushButton" name="pushButton_12">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>30</y>
                <width>182</width>
                <height>25</height>
               </rect>
              </property>
              <property name="text">
               <string>distribute</string>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources/icons/rom_icons.qrc"/>
  <include location="resources/icons/rom_icons.qrc"/>
 </resources>
 <connections/>
</ui>
